src/test/groovy/au/org/ala/merit/ImportServiceSpec.groovy

```diff
@@ -87,7 +87,6 @@ class ImportServiceSpec extends Specification implements ServiceUnitTest<ImportS
 
         then:
         1 * projectService.update('', _) >> { id, details -> projectDetails = details; [resp:[projectId:projectId]]}
-        1 * projectService.generateProjectStageReports(projectId, new ReportGenerationOptions())
```

Comment:
Could change that to a 0 * projectService.generateProjectStageReports to assert that it doesn't happen rather than deleting?  Although that may be confusing in the future once this is the normal system behaviour?